modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectFieldDeleteInfoMVCResourceCommand.java

```diff
@@ -64,7 +64,7 @@ protected void doServeResource(
 				() -> _shouldDeleteObjectFieldObjectValidationRuleSetting(
 					objectField)
 			).put(
-				"showDeletionModal",
+				"showObjectFieldDeletionConfirmationModal",
```

Comment:
Some places I see the delete action being called "delete" and in others "deletion"; I think that from what we have been working on is more common to see "delete", so don't you think its better to change it so it keeps the code uniform in this sense? What do you think?

Reply:
If you think this can be done on a task of a technical debt works for me too, but I think we should change the deletion for delete in all places that is using like this (unless this was discussed to be "deletion" in the past and I'm unaware of this decision)